src/Routes/SeasonsController.ts

```diff
@@ -16,7 +25,24 @@ export class SeasonsController extends BaseController {
                 BaseController._OnError(ex, res);
             }
         });
-
+/**
+ * @swagger
+ *
+ * /[seasonId]:
+ *   get:
+ *     description: Returns a list of Structs in a Season
```

Comment:
Returns the details of the given season id

Reply:
done